docs/src/pages/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -96,20 +61,23 @@ function CheckoutContent() {
         position="absolute"
         color="default"
         elevation={0}
-        className={classes.appBar}
+        sx={{
+          position: 'relative',
+          borderBottom: (theme) => `1px solid ${theme.palette.divider}`,
+        }}
       >
         <Toolbar>
           <Typography variant="h6" color="inherit" noWrap>
             Company name
           </Typography>
         </Toolbar>
       </AppBar>
-      <Container component="main" className={classes.main} maxWidth="sm">
-        <Paper className={classes.paper} variant="outlined">
+      <Container component="main" maxWidth="sm" sx={{ mb: 4 }}>
+        <Paper variant="outlined" sx={{ my: [3, 6], p: [2, 3] }}>
```

Comment:
This is not correct:

```jsx
sx={{ my: [3, 6], p: [2, 3] }}
```

is not equivalent to: 

```jsx
  paper: {
    marginTop: theme.spacing(3),
    marginBottom: theme.spacing(3),
    padding: theme.spacing(2),
    [theme.breakpoints.up('md')]: {
      marginTop: theme.spacing(6),
      marginBottom: theme.spacing(6),
      padding: theme.spacing(3),
    },
  },
```

it's equivalent to 

```jsx
  paper: {
    marginTop: theme.spacing(3),
    marginBottom: theme.spacing(3),
    padding: theme.spacing(2),
    [theme.breakpoints.up('sm')]: {
      marginTop: theme.spacing(6),
      marginBottom: theme.spacing(6),
      padding: theme.spacing(3),
    },
  },
```

Reply:
Ups! I skipped the media query sequence. I will correct it.